app/config/parameters.yml.dist

```diff
@@ -27,10 +27,13 @@ parameters:
 
     domain_name: https://your-wallabag-url-instance.com
 
-    mailer_transport: smtp
-    mailer_host: 127.0.0.1
-    mailer_user: ~
-    mailer_password: ~
+    mailer_transport:  smtp
+    mailer_user:       ~
+    mailer_password:   ~
+    mailer_host:       127.0.0.1
+    mailer_port:       false
```

Comment:
Shouldn't this be `~` instead of `false`?

Reply:
According to the documentation example (Full Default Configuration) this is correct. As port was never provided I can change it to null ~ if you want.

https://symfony.com/doc/current/reference/configuration/swiftmailer.html